Exercises/1-power.test

```diff
@@ -25,14 +25,22 @@
   name: 'square',
   length: [10, 25],
   cases: [
+    [5, 25],
+    [6, 36],
+    [9, 81],
   ]
 }, {
-  name: 'cube',
+  name: 'powerOfThree',
```

Comment:
Why?

Reply:
@tshemsedinov Maybe I got it wrong. I think function `cube` cannot be obtained from [Math.pow(base, exponent)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow) using [bind()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) - the first argument can be binded - so the base will be fixed, not the exponent (number 3 for cube). Thank you.

Reply:
Also [expectation for function `cube`](https://github.com/HowProgrammingWorks/PartialApplication/blob/master/Exercises/1-power.test#L42-L44) don't match to it's name.